Commands/Administration/serverbackup.js

```diff
@@ -14,16 +14,30 @@ const
       emojis: backup.emojis?.length ?? 0,
       stickers: backup.stickers?.length ?? 0
     }
-    : null;
+    : undefined;
 
 /**
- * @this {import('discord.js').BaseInteraction}
+ * @this {GuildInteraction}
  * @param {object}backup*/
 function checkPerm(backup) {
   const creator = backup?.metadata?.[this.guild.db.serverbackup?.allowedToLoad ?? this.client.defaultSettings.serverbackup.allowedToLoad];
   return Array.isArray(creator) ? creator.includes(this.user.id) : creator == this.user.id;
 }
 
+/**
+ * @this {GuildInteraction}
+ * @param {EmbedBuilder}embed
+ * @param {lang}lang*/
+function createProxy(embed, lang) {
+  return new Proxy({ status: undefined }, {
+    set(obj, prop, value) {
+      obj[prop] = value;
+      this.editReply({ embeds: [embed.setDescription(lang(value))] });
```

Comment:
why does that even work (does it?) Shouldn't `this` get overwritten by the set method?

Reply:
fixed that in 22c0c73